File: uts/UT3/acceso a variables de entorno/env.cpp
#include <iostream>
#include <cstdlib>
 
int main()
{
   std::cout << "USER: " << std::getenv("USER") << '\n';
   std::cout << "UNA_VARIABLE: " << std::getenv("UNA_VARIABLE") << '\n';
   std::cout << "OTRA_VARIABLE: " << std::getenv("OTRA_VARIABLE") << '\n';
}

File: uts/UT3/acceso a variables de entorno/Env.java
public class Env {
    public static void main(String[] args) {
        System.out.println("USER: " + System.getenv("USER"));
        System.out.println("UNA_VARIABLE: " + System.getenv("UNA_VARIABLE"));
        System.out.println("OTRA_VARIABLE: " + System.getenv("OTRA_VARIABLE"));
    }
}
